designer/client/src/components/graph/node-modal/tests/TestResults.tsx

```diff
@@ -29,10 +29,7 @@ export default function TestResults({ nodeId }: { nodeId: NodeId }): JSX.Element
             {Object.keys(results.testResultsToShow.context.variables)
                 .sort((a, b) => a.localeCompare(b))
                 .map((key, ikey) => (
```

Comment:
Perhaps we can improve readability by renaming `ikey` to `index`, which technically is the second map argument.

Reply:
Done, changed to `index`!